apps/minifront/src/components/staking/account/delegations.tsx

```diff
@@ -23,17 +24,24 @@ export const Delegations = () => {
 
   return (
     <div className='mt-8 flex flex-col gap-8'>
-      {delegations.map(delegation => (
-        <DelegationValueView
-          key={bech32IdentityKey(getIdentityKeyFromValueView(delegation))}
-          valueView={delegation}
-          unstakedTokens={unstakedTokens}
-          votingPowerAsIntegerPercentage={getVotingPowerAsIntegerPercentage(
-            votingPowerByValidatorInfo,
-            delegation,
-          )}
-        />
-      ))}
+      <AnimatePresence>
+        {delegations.map(delegation => (
+          <motion.div
+            key={bech32IdentityKey(getIdentityKeyFromValueView(delegation))}
+            layout
+            className='bg-charcoal'
```

Comment:
Gave it a background so that text doesn't overlap other text during the animations